The default path for GNU programs in Solaris is /usr/gnu/bin which is different
from what hg-git assumes. Change via test environment is not enough because the
test runner is doing some additional transformations with PATH.

This is a Solaris specific change not intended for upstream.

--- hg-git-1.2.0/tests/run-tests.py
+++ hg-git-1.2.0/tests/run-tests.py
@@ -3266,7 +3266,7 @@ class TestRunner(object):
             os.sep.encode('ascii'), b'/'
         )
         sepb = _sys2bytes(os.pathsep)
-        path = [self._bindir, runtestdir] + osenvironb[b"PATH"].split(sepb)
+        path = [b'/usr/gnu/bin', self._bindir, runtestdir] + osenvironb[b"PATH"].split(sepb)
         if os.path.islink(__file__):
             # test helper will likely be at the end of the symlink
             realfile = os.path.realpath(fileb)
